404.html

```diff
@@ -6,8 +6,13 @@
 <div class="row">
   <div class="small-12 columns">
      <h1>We could not find that page</h1>
-     <article>
-     Please check that the link you used is correct.
-     </article>
+     <p>
+     <h3>If that address is correct, here are three other things you can try:</h3>
+     <ul>
+       <li>Try again later.</li>
+       <li>Check your network connection.</li>
+       <li>If you are connected but behind a firewall, check that your browser has permission to access the Web.</li>
```

Comment:
Thank you, but none of these will actually help the user since if they can see our 404 page they have a working internet and network connection and trying again later won't make the page appear :)